site/content/en/docs/overview/_index.md

```diff
@@ -15,6 +15,8 @@ You can install Kueue on top of a vanilla Kubernetes cluster. Kueue does not rep
 * Compute resources are elastic and can be scaled up and down.
 * Compute resources are heterogeneous (in architecture, availability, price, etc.).
 
+![Theory of Operation](/images/theory-of-operation.svg)
```

Comment:
Add a subtitle for it and maybe some quick paragraph talking about the actors

Reply:
For the subtitle: how does `Kueue operations` or `Kueue operations from different personas` sound?

Reply:
Also if we are going to talk about the different actors should we just not move it to the tasks section where we have definitions of batch admin and user?

Reply:
`Theory of operation` should be good. The paragraph itself can mention the personas and components.

It think it's still useful to have it in the "Overview". You can also add a link to Tasks to learn more about personas.